paths/graphql/05-another-query.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Use a Query to Find Specific Information
+permalink: /graphql/another-query
+next-page: /graphql/mutation
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  Let's build another query. We'll ask GitHub for the ID of an issue so that we can post to that issue using the API in a later step.
+  
+  ![img](../images/gifs/graphql/get-issue-id.gif)
+
+  1. Build a query to find [this issue](https://github.com/githubschool/graph-ql/issues/1)'s unique identifier, or `id`.
```

Comment:
Why this issue? I think that could be valuable context here. Could they post it to any issue?